src/python/pants/backend/terraform/lint/validate/validate_integration_test.py

```diff
@@ -123,6 +124,8 @@ def test_mixed_sources(rule_runner: RuleRunner) -> None:
     lint_results = run_terraform_validate(rule_runner, [target])
     assert len(lint_results) == 1
     assert lint_results[0].exit_code == 1
+    print(f"stdout:\n{lint_results[0].stdout}")
```

Comment:
This is debugging code that will remain until decision made on the design issue described in the comments.